lib/moonchild.js

```diff
@@ -1,16 +1,24 @@
 'use strict';
 
-var _ = require('underscore'),
-    parser = require('./metadata'),
-    estraverse = require('estraverse'),
-    expanders = require('../third_party/expanders');
+var _                = require('underscore');
```

Comment:
Lined up declarations are so much easier to quickly scan with your eyes though, it takes some time to get used to though.

I'll change them back then.